notebooks/diagnostics/cice/advanced.ipynb

```diff
@@ -519,13 +519,13 @@
+       "    title:              POP_gx1v7 grid</pre><div class='xr-wrap' style='display:none'><div class='xr-header'><div class='xr-obj-type'>xarray.Dataset</div></div><ul class='xr-sections'><li class='xr-section-item'><input id='section-02478df6-cc76-42c5-99a3-667fc46733d2' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-02478df6-cc76-42c5-99a3-667fc46733d2' class='xr-section-summary'  title='Expand/collapse section'>Dimensions:</label><div class='xr-section-inline-details'><ul class='xr-dim-list'><li><span class='xr-has-index'>time</span>: 1980</li><li><span>nj</span>: 384</li><li><span>ni</span>: 320</li><li><span>nc</span>: 5</li></ul></div><div class='xr-section-details'></div></li><li class='xr-section-item'><input id='section-0ef4ba92-6128-48cd-bbc7-ab842b97fe75' class='xr-section-summary-in' type='checkbox'  checked><label for='section-0ef4ba92-6128-48cd-bbc7-ab842b97fe75' class='xr-section-summary' >Coordinates: <span>(2)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><ul class='xr-var-list'><li class='xr-var-item'><div class='xr-var-name'><span class='xr-has-index'>time</span></div><div class='xr-var-dims'>(time)</div><div class='xr-var-dtype'>object</div><div class='xr-var-preview xr-preview'>1850-02-01 00:00:00 ... 2015-01-...</div><input id='attrs-c224f455-af25-4853-90a7-8724b1cf570a' class='xr-var-attrs-in' type='checkbox' ><label for='attrs-c224f455-af25-4853-90a7-8724b1cf570a' title='Show/Hide attributes'><svg class='icon xr-icon-file-text2'><use xlink:href='#icon-file-text2'></use></svg></label><input id='data-dddbcc7c-01cf-43c1-8f3e-49d4adbbd754' class='xr-var-data-in' type='checkbox'><label for='data-dddbcc7c-01cf-43c1-8f3e-49d4adbbd754' title='Show/Hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-var-attrs'><dl class='xr-attrs'><dt><span>long_name :</span></dt><dd>model time</dd><dt><span>bounds :</span></dt><dd>time_bounds</dd></dl></div><div class='xr-var-data'><pre>array([cftime.DatetimeNoLeap(1850, 2, 1, 0, 0, 0, 0, has_year_zero=True),\n",
```

Comment:
I don't understand all of this. Is this just HTML formatting?

Reply:
It looks like a few of the id's are different. I think it's just formatting of the output cells since I re-ran them, but I didn't make any actual changes.

Reply:
I had been clearing the cell output for saving. Looks like I can't close the PR, so can you do that?

Reply:
No, I don't have permissions to do so.